backend/core/Extensions/StringExtensions.cs

```diff
@@ -36,6 +36,11 @@ public static bool IsProduction(this string env)
             return env != null && env.Equals("Production", StringComparison.OrdinalIgnoreCase);
         }
 
+        /// <summary>
+        /// Formats the postal code with a space in the middle.
+        /// </summary>
+        /// <param name="postal">The un-formatted postal code.</param>
+        /// <returns>Postal with format XXX XXX.</returns>
         public static string FormatAsPostal(this string postal)
         {
             return postal.Insert(3, " ");
```

Comment:
I'd like to see some checking before adding a space. Perhaps consider checking that there's no space before adding it?
Otherwise, we might run into things like:

```cs
var postal = 'V8V8V8'

var formatted = postal.FormatAsPostal();  // A1A[space]1A1
var oopsIforgotItWasAlreadyFormatted = formatted.FormatAsPostal(); // A1A[space][space]1A1
... // and so on if I keep calling format as postal
```